bazel/repository_locations.bzl

```diff
@@ -1,8 +1,10 @@
 REPOSITORY_LOCATIONS = dict(
     envoy = dict(
-        # envoy 1.26.4
-        commit = "cfa32deca25ac57c2bbecdad72807a9b13493fc1",
-        remote = "https://github.com/envoyproxy/envoy",
+        # envoy 1.26.4 forked with extproc changes
+        # sourced from release/v1.26-backportedfork
+        # should go back to upstream once 1.27 or whever the associated prs are merged
```

Comment:
1.27 is released so these will be in 1.28 at best

Reply:
updated